catalog/CSIC/CSIC_SPEI_2_9.jsonnet

```diff
@@ -1,5 +1,5 @@
-local id = 'CSIC/SPEI/2_8';
```

Comment:
could you open a bug in https://b.corp.google.com/issues?q=status:(open%20%7C%20new%20%7C%20assigned%20%7C%20accepted)%20componentid:1161653&p=1 for tracking this?

Note that we are now more open to adding to the catalog datasets directly from source asset paths (under projects/speibase in your case). Would you like to do that?